package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "defi-guardians",
+  "name": "defi-circuitBreakers",
```

Comment:
Maybe we should name it just `defi-circuit-breaker`?

Reply:
Makes sense! I think it was a ctrl+f replace error when renaming